src/main/java/core/basesyntax/exception/DaoAnnotationNotFound.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class DaoAnnotationNotFound extends Exception {
```

Comment:
**why IOException? (KseniiaMakarova asked)**
_my answer: 
I totally agree it isn't correct_
I fixed that